lib/tic_tac_toe/presenter.ex

```diff
@@ -2,22 +2,45 @@ defmodule TicTacToe.Presenter do
   alias TicTacToe.BoardInspect
 
   def present(board) do
-    space = " "
-
-    """
-
-     #{BoardInspect.get(board, 1) || space} | #{
-      BoardInspect.get(board, 2) || space
-    } | #{BoardInspect.get(board, 3) || space} 
-    ---+---+---
-     #{BoardInspect.get(board, 4) || space} | #{
-      BoardInspect.get(board, 5) || space
-    } | #{BoardInspect.get(board, 6) || space} 
-    ---+---+---
-     #{BoardInspect.get(board, 7) || space} | #{
-      BoardInspect.get(board, 8) || space
-    } | #{BoardInspect.get(board, 9) || space} 
-
-    """
+    {number_of_positions, row_length} = BoardInspect.size(board)
+
+    displayed_board =
+      Enum.to_list(1..number_of_positions)
+      |> Enum.map(fn x ->
+        value = BoardInspect.get(board, x) || " "
+
+        cond do
```

Comment:
This method is pretty long, and is more than five lines. Can you refactor?

Also, I wanted to point out that `cond` is essentially another form of `if`. Much of the time you can leverage pattern matching to achieve the same thing in a way that is a little more declarative and is a bit more conventional in Elixir. For example, you could use `case` here instead. 

I don't care if you use `if` `cond` over pattern matching. I just wanted to point that out to you.

Reply:
Will refactor